src/details/ArborX_DetailsTreeTraversal.hpp

```diff
@@ -54,14 +53,14 @@ struct TreeTraversal<BVH, Predicates, Callback, SpatialPredicateTag>
     else if (bvh_.size() == 1)
     {
       Kokkos::parallel_for(
-          ARBORX_MARK_REGION("BVH:spatial_queries_degenerated_one_leaf_tree"),
+          "ArborX::TreeTraversal::spatial_queries_degenerated_one_leaf_tree",
```

Comment:
Thought about suggesting adding `BVH` here, but then, tree traversal is pretty universal and should work with other trees, so maybe not.